cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -299,6 +301,44 @@ def _add_resources(self):
                 cw_log_group_name=self.log_group.log_group_name if self.config.is_cw_logging_enabled else None,
             )
 
+            self._add_alarms()
+
+    def _add_alarms(self):
+        self.alarms = []
+
+        metrics_for_alarms = {
+            "Mem": cloudwatch.Metric(
+                namespace="CWAgent",
+                metric_name="mem_used_percent",
+                dimensions_map={"InstanceId": self.head_node_instance.ref},
+                statistic="Maximum",
+                period=Duration.seconds(60),
+            ),
+            "Disk": cloudwatch.Metric(
+                namespace="CWAgent",
+                metric_name="disk_used_percent",
+                dimensions_map={"InstanceId": self.head_node_instance.ref},
+                statistic="Maximum",
+                period=Duration.seconds(60),
+            ),
+        }
+
+        for metric_key, metric in metrics_for_alarms.items():
+            alarm_id = f"Pcluster{metric_key}Alarm"
+            alarm_name = f"{metric_key}Alarm_{self.stack.stack_name}_{self.head_node_instance.ref}"
```

Comment:
Instead of having {self.head_node_instance.ref} in the alarm name, I think it would be nice to replace it with "HeadNode". Considering in the future we may want to add alarms for Computes, what do you think?